contracts/RewardsSource.sol

```diff
@@ -103,6 +104,11 @@ contract RewardsSource is Governable {
                 break; // No future slope could match
             }
         }
+
+        // When previewing or sending rewards, check it's own OGV balance,
+        // and if present, send that along as part of the rewards
+        total += Mintable(ogv).balanceOf(address(this));
```

Comment:
This method of handling buyback funds is wrong. It's inside a method that calculates the inflation.

This means that every time inflation is done (every stake / unstake / collect), the inflation is too much by the sum of all buyback funds ever sent to the rewards contract. The OGV sent by the buyback contract never leaves the rewards contract, instead just staying to inflate the mints.

Reply:
Ah, yes. Sorry I totally missed the `collectRewards()` method. Have updated the code. Thanks for catching this